db/changes/131_replace_company_municipality_name_and_number_with_id.sql

```diff
@@ -0,0 +1,13 @@
+-- Add field for municipality ID in company table.
+ALTER TABLE company
+ADD CompanyMunicipalityID INT(11);
+
+-- Populate this field with values from kommune table, based on current CompanyMunicipality number in company table.
+UPDATE company AS c INNER JOIN kommune AS k
+ON k.KommuneNumber = c.CompanyMunicipality
+SET c.CompanyMunicipalityID = k.KommuneID;
```

Comment:
@stoivo Only way this could fail is that the user punched in some municipality that is not in the list of municipalities already and will not be found. That way we could lose some information.